gbfs.md

```diff
@@ -345,11 +328,17 @@ The following fields are all attributes within the main `data` object for this f
 Field Name | REQUIRED | Type | Defines
 ---|---|---|---
 `system_id` | Yes | ID | This is a globally unique identifier for the vehicle share system. Each distinct system or geographic area in which vehicles are operated MUST have its own unique `system_id`. It is up to the publisher of the feed to guarantee uniqueness and MUST be checked against existing `system_id` fields in  [systems.csv](https://github.com/NABSA/gbfs/blob/master/systems.csv) to ensure this. This value is intended to remain the same over the life of the system. <br><br> System IDs SHOULD be recognizable as belonging to a particular system as opposed to random strings - for example, `bcycle_austin` or `biketown_pdx`.
-`language` | Yes | Language | The language that will be used throughout the rest of the files. It MUST match the value in the [gbfs.json](#gbfsjson) file.
-`name` | Yes | String | Name of the system to be displayed to customers.
-`opening_hours`<br/>*(added in v3.0-RC)*| Yes | String | Hours and dates of operation for the system in [OSM opening_hours](https://wiki.openstreetmap.org/wiki/Key:opening_hours) format. *(added in v3.0-RC)*
-`short_name` | OPTIONAL | String | OPTIONAL abbreviation for a system.
-`operator` | OPTIONAL | String | Name of the system operator.
+`languages` | Yes | Array | List of languages used in translated strings. Each element in the list must be of type Language.
```

Comment:
That is a very valid point, since "Array" is defined in the spec as "A JSON element consisting of an ordered sequence of zero or more values". I'm curious if that's intentional. Is there any field in the spec that would carry different meanings for an empty array as opposed to omitting the field altogether? If not then maybe the right thing to do is to require (optional?) arrays to have at least one element?

Reply:
Maybe this could be clarified in the `languages` array definition but wouldn't there always be at least one value in the array? For example if you don't offer any translations and only offer feeds in `en` then that's the array value.

Reply:
Yes exactly - there should always be at least one value, but the spec leaves this open. I.e. providing an empty array would still be valid according to the spec.